server/routes/shared/showReferralPresenter.ts

```diff
@@ -75,6 +77,48 @@ export default class ShowReferralPresenter {
         ]
   }
 
+  get responsibleOfficersDetails(): SummaryListItem[][] {
+    return this.responsibleOfficers.map(responsibleOfficer => {
+      const { staff } = responsibleOfficer
+
+      if (!staff) {
+        // This is a temporary step - I want to not break the page if fields are missing, most likely with dev data.
+        // This will hopefully be removed when we switch to using a dedicated Responsible Officer endpoint.
+        return [
+          {
+            key: 'Name',
+            lines: ['Not found'],
+          },
+          {
+            key: 'Phone',
+            lines: ['Not found'],
+          },
+          {
+            key: 'Email address',
+            lines: ['Not found'],
+          },
+        ]
+      }
+
+      const name = `${staff.forenames || ''} ${staff.surname || ''}`.trim() || 'Not found'
```

Comment:
nit: can we condense this to one block using `?.` on the staff object?

Reply:
I did try that originally, it turns out `${undefined}` === 'undefined', so if someone was missing a first or last name it would be displayed as "Peter undefined" or the other way round 🤦. 

This implementation does feel a bit over the top though, so let me know if you can think of any improvements

Reply:
lol! doesn't `${undefined} || ''` return empty string though?